### PR TITLE
alpha main menu

### PR DESCRIPTION
**Pull request by echochase**
** CHANGES - PLEASE REVIEW SINCE ITS MY FIRST TIME N I DONT CODE TOO WELL **

- added resources (button, mainmenu background)
- added menubutton
- added style files

anyway not much but still i dont wanna pollute w bad code

### CODE REVIEW

File: client/src/App.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#root {
```
Comment (Suchpuns) [inline client/src/App.css:1]: Feel free to just delete this instead of commenting it out

File: client/src/App.tsx [hunk 2]
```diff
@@ -1,35 +1,15 @@
-import { useState } from 'react'
```
Comment (Suchpuns) [inline client/src/App.tsx:1]: You don't want this in App.tsx
make a Menu.tsx and put your code there.
then put Menu.tsx inside App.tsx to test it out.

Comment (Suchpuns) [reply client/src/App.tsx:1]: main reason is cuz we are gonna have other routes 
say /game or /room
So App.tsx is gonna need to have the components for each of those routes.
if that didn't make much sense lemme know and I can show you.

**Review by Suchpuns — CHANGES_REQUESTED** [review]

File: client/src/App.tsx (hunk 2 above)
Comment (Suchpuns) [reply client/src/App.tsx:1]: MainMenu.tsx*

File: client/src/Styles/MainMenu.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.menu-container {
+    display: flex; /* apparently this makes the children components centered*/
+    flex-direction: column; /* Set to row to align buttons horizontally */
+    justify-content: center;
+    align-items: center;
+    background-image: url("../Resources/Other/MainMenu.png");
+    background-size: auto 100%;
+    height: 100vh;
+    width: 100vw;
```
Comment (Suchpuns) [inline client/src/Styles/MainMenu.css:10]: add
background-position: center;

to actually center the background

File: client/src/Resources/Other/MainMenu.png
Comment (Suchpuns) [inline client/src/Resources/Other/MainMenu.png:1]: logo is not centred

Comment (Suchpuns) [reply client/src/Resources/Other/MainMenu.png:1]: also increase the width, its not long enough for my laptop screen 💀

Comment (echochase) [reply client/src/Resources/Other/MainMenu.png:1]: yepyep leaving this till after we decide on logo

**Review by Suchpuns — APPROVED** [review]
LGTM

**Pull request merged**